website/content/en/preview/provisioner.md

```diff
@@ -54,9 +54,15 @@ spec:
   # These requirements are combined with pod.spec.affinity.nodeAffinity rules.
   # Operators { In, NotIn } are supported to enable including or excluding values
   requirements:
-    - key: "node.kubernetes.io/instance-type"
+    - key: "karpenter.k8s.aws/instance-category"
```

Comment:
Do we want to leak AWS-specific labels into the generic provisioners docs?

Reply:
Maybe there is a discussion worth having about making `instance-category` a generic Karpenter concept?

Reply:
> generic provisioners docs?

The rough direction I've been discussing with @chrisnegus is that the website will be AWS specific. We may spin up a second neutral website for karpenter-core, but I think the docs quality hit that we take for being neutral is too high.